[PATCH] make setregid02 work

Upstream-Status: Inappropriate [configuration]

there is no "nobody" group in oe-core, the user "nobody" belongs to
"nogroup" group, so replace nobody with nogroup to make the test pass

Signed-off-by: Roy.Li <rongqing.li@windriver.com>
---
 testcases/kernel/syscalls/setregid/setregid02.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/testcases/kernel/syscalls/setregid/setregid02.c b/testcases/kernel/syscalls/setregid/setregid02.c
index 8058627..866bee4 100644
--- a/testcases/kernel/syscalls/setregid/setregid02.c
+++ b/testcases/kernel/syscalls/setregid/setregid02.c
@@ -41,7 +41,7 @@ static gid_t neg_one = -1;
 
 static struct passwd *ltpuser;
 
-static struct group nobody, root, bin;
+static struct group nogroup, root, bin;
 
 /*
  * The following structure contains all test data.  Each structure in the array
@@ -57,17 +57,17 @@ struct test_data_t {
 	char *test_msg;
 } test_data[] = {
 	{
-	&neg_one, &root.gr_gid, EPERM, &nobody, &nobody,
+	&neg_one, &root.gr_gid, EPERM, &nogroup, &nogroup,
 		    "After setregid(-1, root),"}, {
-	&neg_one, &bin.gr_gid, EPERM, &nobody, &nobody,
+	&neg_one, &bin.gr_gid, EPERM, &nogroup, &nogroup,
 		    "After setregid(-1, bin)"}, {
-	&root.gr_gid, &neg_one, EPERM, &nobody, &nobody,
+	&root.gr_gid, &neg_one, EPERM, &nogroup, &nogroup,
 		    "After setregid(root,-1),"}, {
-	&bin.gr_gid, &neg_one, EPERM, &nobody, &nobody,
+	&bin.gr_gid, &neg_one, EPERM, &nogroup, &nogroup,
 		    "After setregid(bin, -1),"}, {
-	&root.gr_gid, &bin.gr_gid, EPERM, &nobody, &nobody,
+	&root.gr_gid, &bin.gr_gid, EPERM, &nogroup, &nogroup,
 		    "After setregid(root, bin)"}, {
-	&bin.gr_gid, &root.gr_gid, EPERM, &nobody, &nobody,
+	&bin.gr_gid, &root.gr_gid, EPERM, &nogroup, &nogroup,
 		    "After setregid(bin, root),"}
 };
 
@@ -165,7 +165,7 @@ static void setup(void)
 } while (0)
 
 	GET_GID(root);
-	GET_GID(nobody);
+	GET_GID(nogroup);
 	GET_GID(bin);
 
 	TEST_PAUSE;
-- 
1.9.1

